scripts/vet.sh

```diff
@@ -67,6 +67,9 @@ not git grep "\"github.com/golang/protobuf/*" -- "*.go" ':(exclude)reflection/te
 # - Ensure all usages of grpc_testing package are renamed when importing.
 not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" -- "*.go"
 
+# Ensure that no trailing spaces are found.
```

Comment:
I'm not sure why, exactly, but all the section comments start with `-`; please continue the pattern here.